src/optimize/dynamic_dll_plugin/dynamic_dll_plugin.js

```diff
@@ -247,19 +206,14 @@ export class DynamicDllPlugin {
   registerDoneHook(compiler) {
     compiler.hooks.done.tapPromise('DynamicDllPlugin', async stats => {
       if (stats.compilation.needsDLLCompilation) {
-        // Logic to run the max compilation requirements.
-        // Only enable this for CI builds in order to ensure
-        // we have an healthy dll ecosystem.
-        if (IS_KIBANA_DISTRIBUTABLE && (this.performedCompilations === this.maxCompilations)) {
-          throw new Error(
-            'All the allowed dll compilations were already performed and one more is needed which is not possible'
-          );
-        }
-
         // Run the dlls compiler and increment
         // the performed compilations
-        await this.runDLLCompiler(compiler);
-        this.performedCompilations++;
+        try {
```

Comment:
Unnecessary try/catch

Reply:
@spalger yeah I know, however I found that the errors reaching this try catch were not logged correctly, we were getting something like:

`[fatal] [object Object]`

It might have to do with a bug fix merged on webpack for v4.29.3: `WebpackErrors now console.log correctly in node > 10`. I'll keep the try catch for now, and I've added a note to recheck it after upgrading webpack 😃

Reply:
Looks like you're using the error now so 👍